staging/src/k8s.io/apiserver/pkg/endpoints/filters/authentication.go

```diff
@@ -98,6 +101,9 @@ func WithAuthentication(handler http.Handler, auth authenticator.Request, failed
 		if len(apiAuds) > 0 {
 			req = req.WithContext(authenticator.WithAudiences(req.Context(), apiAuds))
 		}
+		ctx := req.Context()
+		ae := request.AuditEventFrom(ctx)
```

Comment:
Nit: prefer inlining
```suggestion
		ae := request.AuditEventFrom(req.Context())
```